storage/entities/base_entity.js

```diff
@@ -175,13 +175,12 @@ class BaseEntity {
 		);
 	}
 
-	getValuesSet(data) {
-		return `(${this.adapter.parseQueryComponent(
-			Object.keys(data)
-				.map(key => this.fields[key].serializeValue(data[key], 'insert'))
-				.join(','),
-			data
-		)})`;
+	getValuesSet(data, attributes = []) {
```

Comment:
You can't set `attribute` default value as empty array. If you do, `_getValueSetForObject` will map over the empt array and will never use `Object.keys(data)`